assets/js/ajax_search_select.js

```diff
@@ -3,6 +3,10 @@ $(function() {
 
   $('.search-select').aplusSearchSelect();
   $('.search-select-ajax').aplusSearchSelectAjax();
+
+  jQuery.expr[':'].icontains = function(a, i, m) {
+    return jQuery(a).text().toUpperCase().indexOf(m[3].toUpperCase()) >= 0;
+  };
```

Comment:
Is it possible to give more descriptive parameter names for `a`, `i`, and `m`?
Right now, I don't know what they mean just from looking at the code.

Reply:
Yes, updated the code.